server/src/main/java/org/elasticsearch/index/mapper/MapperService.java

```diff
@@ -813,11 +818,7 @@ public void close() throws IOException {
      */
     @Deprecated
     public static boolean isMetadataFieldStatic(String fieldName) {
-        if (IndicesModule.getBuiltInMetadataFields().contains(fieldName)) {
-            return true;
-        }
-        // if a node had Size Plugin installed, _size field should also be considered a meta-field
-        return fieldName.equals("_size");
+        return META_FIELDS_BEFORE_7_8.contains(fieldName);
```

Comment:
If I remember correctly we only use `isMetadataFieldStatic` in a few places now where we read from older nodes. Should we simply make the collection public and remove the static method? Or rename method to something like "isMetaFieldBefore7dot8" or something, just to improve readability in the locations where this it is actually used.

Reply:
@cbuescher Thanks for the feedback! I have removed `isMetadataFieldStatic` method, and made the collection public.